react-native/package.json

```diff
@@ -14,14 +12,14 @@
   },
   "dependencies": {
     "buffer": "^3.5.1",
-    "framed-msgpack-rpc": "https://github.com/keybase/node-framed-msgpack-rpc.git#nojima/keybase-client-changes",
+    "framed-msgpack-rpc": "keybase/node-framed-msgpack-rpc#nojima/keybase-client-changes",
     "iced-runtime": "^1.0.3",
     "immutable": "^3.7.5",
     "lodash": "^3.10.1",
     "moment": "^2.10.6",
-    "purepack": "https://github.com/keybase/purepack.git#nojima/keybase-client-changes",
+    "purepack": "keybase/purepack#nojima/keybase-client-changes",
     "qrcode-generator": "^1.0.0",
-    "react-native": "git://github.com/keybase/react-native#packager-allows-mobile-files",
+    "react-native": "keybase/react-native#packager-allows-mobile-files-0-14-2",
```

Comment:
using the new npm format (drop the git part) and pointing to a version specific branch. we own a fork to allow the .mobile.js fallbacks in the packager